database/staking_delegations.go

```diff
@@ -27,6 +29,20 @@ func (db *Db) SaveDelegations(delegations []types.Delegation) error {
 		}
 
 		err := db.storeUpToDateDelegations(paramsNumber, delegationSlice)
+		if err != nil {
+			// Get the error code
+			var errorCode string
+			_, scanErr := fmt.Sscanf(err.Error(), utils.ErrNotFound, &errorCode, &errorCode)
+			if scanErr != nil {
+				return fmt.Errorf("error while scanning error: %s", scanErr)
+			}
+
+			// If error is different than delegation not found, we need to return it
+			if errorCode != codes.NotFound.String() {
+				return fmt.Errorf("error while searching for delegations %s", err)
+			}
+		}
+
```

Comment:
Why is this necessary?

Reply:
I added this to handle the error that bdjuno throws when it parses old delegate type tx and can't find it

Reply:
I don't think we should ignore this error honestly. If the validator is not stored, we can't ignore the fact that something is wrong. Something like this should never happen

Reply:
Okay sure! I've tested it locally and I can't reproduce this error anymore so I have removed this part from the code